pkg/selinux/selinux.go

```diff
@@ -361,19 +361,19 @@ func FreeLxcContexts(scon string) {
 	}
 }
 
-func GetLxcContexts() (processLabel string, fileLabel string) {
+func GetLxcContexts() (processLabel string, fileLabel string, err error) {
 	var (
 		val, key string
 		bufin    *bufio.Reader
 	)
 
 	if !SelinuxEnabled() {
-		return "", ""
+		return "", "", nil
 	}
 	lxcPath := fmt.Sprintf("%s/contexts/lxc_contexts", getSELinuxPolicyRoot())
 	in, err := os.Open(lxcPath)
 	if err != nil {
-		return "", ""
+		return "", "", nil
```

Comment:
Don't ask me! Haha

I guess if there's no lxc_contexts then we just return empty. Callers seem to handle this by doing nothing.